acpdr/swagger-specs/schema-registry.yaml

```diff
@@ -1083,13 +1213,13 @@ paths:
       - $ref: '#/parameters/x-sandbox-name'
       - $ref: '#/parameters/accept'
       - name: RESOURCE_ID
-        description: The meta:altId or URL-encoded $id value of the schema, mixin, or data type you want to export.
+        description: The meta:altId or URL-encoded $id value of the schema, field group, or data type you want to export.
         required: true
         in: path
         type: string
       responses:
         '200':
-          description: A successful response returns the details of the exported resource and its dependent mixins and data types. The contents of this payload can then be used as the request body in a POST /rpc/import call to import the resource to the desired sandbox or IMS Organization.
+          description: A successful response returns the details of the exported resource and its dependent field groups and data types. The contents of this payload can then be used as the request body in a POST /rpc/import call to import .the resource to the desired sandbox or IMS Organization.
```

Comment:
I'd backtick `/rpc/import`, since it's an endpoint path.

Reply:
Also seems like "the resource to the desired sandbox or IMS Organization." got left behind here?